.travis.yml

```diff
@@ -59,34 +59,36 @@ jobs:
     # the `platforms` defined in `kitchen.yml`
     # - env: INSTANCE=default-debian-10-master-py3
+    # - env: INSTANCE=default-ubuntu-2004-master-py3
     - env: INSTANCE=default-ubuntu-1804-master-py3
-    - env: INSTANCE=centos-centos-8-master-py3
-    # - env: INSTANCE=default-fedora-31-master-py3
-    - env: INSTANCE=default-opensuse-leap-151-master-py3
-    - env: INSTANCE=default-amazonlinux-2-master-py3
+    - env: INSTANCE=redhat-centos-8-master-py3
+    - env: INSTANCE=redhat-fedora-32-master-py3
+    # - env: INSTANCE=redhat-fedora-31-master-py3
+    - env: INSTANCE=default-opensuse-leap-152-master-py3
+    - env: INSTANCE=redhat-amazonlinux-2-master-py3
+    # - env: INSTANCE=default-debian-10-3000-3-py3
+    # - env: INSTANCE=default-debian-9-3000-3-py3
+    # - env: INSTANCE=default-ubuntu-1804-3000-3-py3
+    # - env: INSTANCE=redhat-centos-8-3000-3-py3
+    # - env: INSTANCE=redhat-centos-7-3000-3-py3
+    # - env: INSTANCE=redhat-fedora-31-3000-3-py3
+    # - env: INSTANCE=default-opensuse-leap-152-3000-3-py3
+    # - env: INSTANCE=redhat-amazonlinux-2-3000-3-py3
+    # - env: INSTANCE=default-ubuntu-1804-3000-3-py2
+    # - env: INSTANCE=default-ubuntu-1604-3000-3-py2
+    # - env: INSTANCE=arch-arch-base-latest-3000-3-py2
     # - env: INSTANCE=default-debian-10-2019-2-py3
     # - env: INSTANCE=default-debian-9-2019-2-py3
     # - env: INSTANCE=default-ubuntu-1804-2019-2-py3
-    # - env: INSTANCE=centos-centos-8-2019-2-py3
-    # - env: INSTANCE=default-fedora-31-2019-2-py3
-    # - env: INSTANCE=default-opensuse-leap-151-2019-2-py3
-    # - env: INSTANCE=centos-centos-7-2019-2-py2
-    # - env: INSTANCE=default-amazonlinux-2-2019-2-py3
-    - env: INSTANCE=arch-arch-base-latest-2019-2-py2
-    # - env: INSTANCE=default-fedora-30-2018-3-py3
-    # - env: INSTANCE=default-debian-9-2018-3-py2
-    # - env: INSTANCE=default-ubuntu-1604-2018-3-py2
-    # - env: INSTANCE=centos-centos-7-2018-3-py2
-    # - env: INSTANCE=default-opensuse-leap-151-2018-3-py2
-    # - env: INSTANCE=default-amazonlinux-1-2018-3-py2
-    - env: INSTANCE=arch-arch-base-latest-2018-3-py2
-    # - env: INSTANCE=default-debian-8-2017-7-py2
-    # - env: INSTANCE=default-ubuntu-1604-2017-7-py2
-    # - env: INSTANCE=centos-centos-6-2017-7-py2
-    # - env: INSTANCE=default-fedora-30-2017-7-py2
-    # - env: INSTANCE=default-opensuse-leap-151-2017-7-py2
-    # - env: INSTANCE=default-amazonlinux-1-2017-7-py2
-    # - env: INSTANCE=arch-arch-base-latest-2017-7-py2
+    # - env: INSTANCE=default-ubuntu-1604-2019-2-py3
+    # - env: INSTANCE=redhat-centos-8-2019-2-py3
+    # - env: INSTANCE=redhat-centos-7-2019-2-py3
+    # - env: INSTANCE=redhat-fedora-31-2019-2-py3
+    # - env: INSTANCE=default-opensuse-leap-152-2019-2-py3
+    # - env: INSTANCE=redhat-amazonlinux-2-2019-2-py3
+    # - env: INSTANCE=redhat-centos-6-2019-2-py2
+    # - env: INSTANCE=redhat-amazonlinux-1-2019-2-py2
+    # - env: INSTANCE=arch-arch-base-latest-2019-2-py2
```

Comment:
While adding the new pre-salted images, found that `arch` simply won't working -- temperamental:

* https://travis-ci.com/github/saltstack-formulas/iscsi-formula/jobs/331884105

Even restarted the job for this PR (before adding the commit) and it failed.  Can be investigated later, if desired.